src/main/java/uk/gov/hmcts/reform/authorisation/exceptions/AbstractAuthorisationException.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.reform.authorisation.exceptions;
+
+import feign.FeignException;
+
+abstract public class AbstractAuthorisationException extends RuntimeException {
```

Comment:
Since the class is already abstract I think we shouldn't need to encode that in the name as well, what do you think?

Reply:
There is a code style rule to have prefix of `Abstract` in class name in case it is flagged as abstract

Reply:
Ok, I see. Maybe it would be worth evaluating those rules at some point and removing the ones which don't bring much value.